sections/06-vibrant-bodies.md

```diff
@@ -0,0 +1,201 @@
+
+_In Passing_ suggests that a work of art can function differently, perhaps, than one might traditionally expect; as more than just a static object for aesthetic contemplation. As with Jack Chambers' embodied technics of _perceptual realism_, it is a system to see (and hear) with, a means to generate and transform experience.
+
+"Perceptual realism incorporates two systems of technology (historic and industrial) and two systems of visibility (body and mind) to structure a reflector-object of experience... two inter-working processes motivate and advance one another intentionally to create this transmitter unit... The closure of these two extending systems... creates its own object. It is an object to see with, rather than a thing just to see" (Chambers 41-2).
+
+While Chambers uses the term "object" here, he is nonetheless suggesting that the work of art can serve as catalyst for an ongoing process of perceptual transformation within everyday experience. In a suitably dark room and with sufficient attention, the continuously shifting, stochastic sparkles of _In Passing_'s digital processing have a way of entering into a viewer's persistent vision, activating a mode of seeing that continues beyond the video's duration. Sparkles can be seen (and imagined) in anything, even with eyes closed. This model shifts the art, by means of an engaged and embodied viewer, beyond the temporal and spatial frame of the video and out into the world. _In Passing_ thus becomes implicated with our own seeing and hearing, a sort of perceptual synthesizer (perhaps this what Chambers means by a "transmitter unit?") that modulates the carrier wave of perception with the work's own oscillations. A system of creative, reciprocal, and active seeing that continues to operate in the world. "My eye... being instrument for striking sparks, is bequeathed visions at every illumination... Similar vistas being available to any viewer willing to release his eye for comparable movement" (Brakhage 26).
+
+Arguably, _In Passing_'s eighteen-minute timespan is only marginally sufficient (if at all) to activate such a "system to see with," simply because its processes make, with time, a material imprint on perception. Nonetheless, its duration, coupled to its highly reduced visual vocabulary, slow pace, absence of narrative, and preponderance of silence, has the tendency to mark or make prominent our embodiment. Even as seated and relatively still cinema-goers, we notice our bodies. "The still body is still a body doing. It is acting and being acted on... the activity of stilling can be considered a choreography of non-acts or counter-movements to the host of budding impulses that develop into drives and ultimately forms of action" (Priest, "Felt as Thought" 47).
```

Comment:
Comment from David: "re: 'Arguably, In Passing's eighteen-minute timespan is only marginally sufficient (if at all) to activate such a "system to see with,'. I think you can take this out and own the claim of the piece. The claim, after all, isn’t that it demands a material transformation, but rather that it is capable of catalyzing one in the sensitive viewer. "